data/layout/telescope_positions-North-TestLayout.ecsv

```diff
@@ -1,26 +1,51 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string, description: common telescope name}
-# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST15 (central MST)}
-# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST15 (central MST)}
-# - {name: pos_z, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
-# meta: !!omap
-# - data_type: positionfile
-# - description: "telescope positions for CTA North (La Palma)"
-# - center_lon: "-17.8920302 deg"
-# - center_lat: "28.7621661 deg"
-# - center_alt: "2177m"
-# - center_northing: "3185066.0 m"
-# - center_easting: "217611.0 m"
-# - corsika_obs_level: "2158 m"
-# - corsika_sphere_radius: {LST: 12.5 m, MST: 9.6 m, SST: 3 m}
-# - corsika_sphere_center: {LST: 16 m, MST: 9 m, SST: 3.25 m}
-# - EPSG: 32628
-# - comments: |
-#       version 0.1
-#       UTM Zone 28
-#       prod5
+# - {name: telescope_name, datatype: string}
+# - {name: pos_x, unit: m, datatype: float64}
+# - {name: pos_y, unit: m, datatype: float64}
+# - {name: pos_z, unit: m, datatype: float64}
+# meta:
+#   EPSG: 32719
+#   center_alt: !astropy.units.Quantity
+#     unit: &id001 !astropy.units.Unit {unit: m}
+#     value: 2150.0
```

Comment:
Why the altitude change?

Reply:
I had copied the entire meta data from other file but this turned out to be wrong. For consistency, I now downloaded `telescope_positions-North-TestLayout` from the DB and copied to data/layout